specification/communication/data-plane/Sms/stable/2021-03-07/communicationservicessms.json

```diff
@@ -3,7 +3,7 @@
   "info": {
     "title": "Azure Communication SMS Service",
     "description": "Azure Communication SMS Service",
-    "version": "2021-03-07"
+    "version": "2022-07-07"
```

Comment:
is it necessary to increase api version taking into account the new parameter looks not a braking change?